applications/models/application_round.py

```diff
@@ -30,6 +30,7 @@ class ApplicationRound(models.Model):
     name: str = models.CharField(max_length=255)
     target_group: str = models.CharField(choices=TargetGroupChoice.choices, max_length=50)
     criteria: str = models.TextField(default="")
+    notes_when_applying: str = models.TextField(default="")
```

Comment:
Should this have `blank=True`, when even the default value is `""` 🤔

Reply:
Yes, otherwise the field is still required everywhere by default. Fixed.